contracts/test/TestJBveBanny.t.sol

```diff
@@ -28,12 +32,15 @@ contract JBveBannyTests is TestBaseWorkflow {
     _jbOperatorStore = jbOperatorStore();
     _jbveTokenUriResolver = jbveTokenUriResolver();
     _jbController = jbController();
+    _redemptionTerminal = jbERC20PaymentTerminal();
+    _jbTerminalToken = jbToken();
```

Comment:
See naming comment in base workflow (better here - paymentToken maybe? or remove the 'jb' prefix?)